spec/requests/prendre_rdv_spec.rb

```diff
@@ -45,9 +45,20 @@
         expect(response.body).to include("Sélectionnez le service avec qui vous voulez prendre un RDV")
       end
 
-      it "shows a hint to help find a rdv with a referent agent in case the user is looking for the service of a follow_up motifs" do
-        get root_path(departement: "75", city_code: "75056", latitude: "48.859", longitude: "2.347", address: "Paris 75001")
-        expect(response.body).to include("Pour prendre un RDV de suivi avec un de vos agents référent")
+      context "when territory has no follow up motifs" do
+        it "does not show a hint to help find a rdv with a referent agent in case the user is looking for follow_up motifs" do
```

Comment:
nitpick total et absolument pas bloquant pour cette PR mais on pourrait traduire ce `it` en français et le simplifier pour la santé d’esprit de toustes :)